wsapi/debugapi.go

```diff
@@ -427,15 +427,16 @@ func runCmd(cmd string) {
 
 	return
 }
+
 func HandleSimControl(state interfaces.IState, params interface{}) (interface{}, *primitives.JSONError) {
-	droprate := new(GetCommands)
-	err := MapToObject(params, droprate)
+	cmdLines := new(GetCommands)
```

Comment:
renamed b/c it didn't look like we are limited to droprate commands here

Reply:
good change